charts/buffer/templates/proxy-tcell-configmap.yaml

```diff
@@ -0,0 +1,24 @@
+{{- if .Values.proxy.tcell.enabled }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: buffer-tcell-configmap
+  labels:
+    {{ template "buffer.labels" . }}
+data:
+  tcell-config: |-
+    {
+      "version": 1,
+      "applications": [
+        {
+          "app_id": "$(TCELL_APP_ID)",
```

Comment:
@mflinn-broad and I figured out that this actually needs the secrets rendered in directly -- not bash variables. `/var/log/apache2/tcell.log` on the container likely has 403 errors indicating that it can't talk out to the tcell server for bad credentials.